compiler/frontend/spark/src/main/java/edu/snu/nemo/compiler/frontend/spark/sql/NemoSparkSQL.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2018 Seoul National University
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package edu.snu.nemo.compiler.frontend.spark.sql;
+
+/**
+ * Common interface for Nemo Spark classes.
+ */
+public interface NemoSparkSQL {
```

Comment:
Please rename this interface. IT is not that informative.

Reply:
Please update the comment also.

Reply:
I've addressed your comment 😄

Reply:
@sanha